templates/init/functions/javascript/package.lint.json

```diff
@@ -3,6 +3,7 @@
   "description": "Cloud Functions for Firebase",
   "scripts": {
     "lint": "./node_modules/.bin/eslint .",
+    "build": "exit 0",
```

Comment:
I'm not sure what build command to run for javascript, but the predeploy script breaks if this is an empty string so I've left this here for now.

Reply:
1. We don't have build anymore so this can be cut.
1. If you want to (more gracefully) support empty strings, you can filter those out in lifecycleHooks.js. This will change the order a bit you can either:
  * Move your noop code until after you change string to string[] and after you `_.reject(commands, '')`
  * Change runCommand to return `Promise.resolve()` if the command is the empty string